services/upload/src/main/java/app/coronawarn/server/services/federation/upload/config/UploadServiceConfig.java

```diff
@@ -100,8 +100,8 @@ public ReportType getDefaultReportType() {
       return defaultReportType;
     }
 
-    public void setDefaultReportType(ReportType defaultReportType) {
-      this.defaultReportType = defaultReportType;
```

Comment:
this isn't required then?

Reply:
It's still required. We just have to consider that `application.yaml` has a number and our class expects a `ReportType`